api.planx.uk/docs/schemas.yaml

```diff
@@ -0,0 +1,18 @@
+openapi: 3.1.0
+info:
+  title: Plan✕ API
+  version: 0.1.0
+paths: {}
+components:
+  schemas:
+    SessionPayload:
+      type: object
+      properties:
+        payload:
+          type: object
+          properties:
+            sessionId:
+              type: string
+      example:
+        payload:
+          sessionId: "123"
```

Comment:
Similar to above, it seems like the "Schemas" section isn't actually rendering on the pizza yet: https://api.2164.planx.pizza/docs/ versus https://api.editor.planx.dev/docs/#/